src/toffy/json_utils.py

```diff
@@ -194,3 +195,40 @@ def check_for_empty_files(bin_file_dir):
 
     # return the list of fov names
     return empty_json_files
+
+
+def check_fov_resolutions(bin_file_dir, run_name):
+    """Use the run metadata to calculate the resolution of each fov
+    Args:
+        bin_file_dir (str): directory containing the run json file
+        run_name (str): name of the run and corresponding run file
+
+    Returns:
+        pd.DataFrame: details fov names and corresponding resolution value
+    """
+    # read in run metadata
+    run_file_path = os.path.join(bin_file_dir, run_name + ".json")
+    io_utils.validate_paths([run_file_path])
+    run_metadata = read_json_file(run_file_path, encoding="utf-8")
+
+    fov_names, resolutions = [], []
+    for fov in run_metadata.get("fovs", ()):
+        # get fov names
+        fov_number = fov.get("runOrder")
+        name = f"fov-{fov_number}-scan-1"
```

Comment:
This would cause an issue if `run_metadata` contains `scan-2` or `scan-3`. In that case is you'd gets something like this:

```
...
fov-n-scan-1: xxx
fov-n-scan-1: xxx # actually fov-n-scan-2's resolution
fov-n-scan-3: xxx # actually fov-n-scan-3's resolution
...
```

The watcher script right now has a similar issue with `runCount` during initialization, however all it does is overwrite static dict keys during initialization, so I left it be. In this case, however, I'd explicitly skip anything that has `scanCount` (or whatever the key is) greater than 1 (or warn the user that anything with scans > 1 will be ignored during extraction).

Reply:
I had thought that multiple scan info is still stored under a single FOV key in the run metadata? If so, that shouldn't be a problem here, but if the run file structure has changed then yeah this could be a problem.

Reply:
Yup just verified with a run file from 7/05 that there is only a single `runOrder`, `frameSizePixels`, and `fovSizeMicrons` key even when there are multiple scans.